src/me/mrCookieSlime/Slimefun/api/TickerTask.java

```diff
@@ -196,56 +202,73 @@ public void info(CommandSender sender) {
 		sender.sendMessage(ChatColor.translateAlternateColorCodes('&', "&6Skipped Machines: &e" + skipped));
 		sender.sendMessage("");
 		sender.sendMessage(ChatColor.translateAlternateColorCodes('&', "&6Ticking Machines:"));
-		
+
+		List<Map.Entry<String, Long>> timings = machineCount.keySet().stream()
+				.map(key -> new AbstractMap.SimpleEntry<>(key, machineTimings.getOrDefault(key, 0L)))
+				.sorted((o1, o2) -> o2.getValue().compareTo(o1.getValue()))
```

Comment:
Again: Couldn't you just use "Map.Entry.comparingByValue" here too?

Reply:
Again, feel free to try it yourself. At least with my javac despite it seeming completly fine the generics casue it to fail. It thinks it's like <String, Object> and then I'd need to cast and then it fails. This works fine regardless

Reply:
Oh sorry, didn't see your previous reponse to it. That's fine then.